core/assets/bundles/bundle_vi.properties

```diff
@@ -520,10 +520,10 @@ filter.option.chance = Tỷ lệ
 filter.option.mag = Độ lớn
 filter.option.threshold = Ngưỡng
 filter.option.circle-scale = Độ lớn vòng tròn
-filter.option.octaves = Octaves
+filter.option.octaves = Quãng tám
```

Comment:
Maybe "octaves" doesnt mean that

Reply:
I dont sure
That just follow Google

Reply:
Pls... dont use Google translate, its BAD

Reply:
I know but that is the most correct answer(i think)